nengo/tests/test_simulator.py

```diff
@@ -83,8 +79,25 @@ def test_usage_in_with_statement(Simulator):
     with pytest.raises(ValueError):
         sim.run(1.)
     with pytest.raises(ValueError):
+        sim.reset()
+
+
+def test_close_steps(RefSimulator):
+    m = nengo.Network()
```

Comment:
Maybe add a docstring explaining that this test is checking the two ways to close a simulator (implicitly with `with` and explicitly with `close()`)?